backend/src/AppServer.hs

```diff
@@ -219,6 +219,25 @@ reprocessReports = do
   forM_ reports processReport
   updateActiveStatus
 
+leaguePoints2025 :: League -> Int -> Int -> [(Int, Int)] -> Double
+leaguePoints2025 league totalWins totalGames stats = baseScore + unplayedMultiplier * unplayedGames
+  where
+    baseScore = 0.1 * fromIntegral totalGames + fromIntegral totalWins
+    winRate = fromIntegral totalWins / fromIntegral totalGames :: Double
+    unplayedMultiplier
+      | (league == GeneralLeague || league == LoMELeague) && totalGames < 14 = 0
+      | (league == GeneralLeague || league == LoMELeague) && totalGames < 24 = 0.5 * winRate
+      | league == GeneralLeague || league == LoMELeague = winRate
+      | totalGames < 8 = 0
+      | totalGames < 18 = 0.5 * winRate
+      | otherwise = winRate
```

Comment:
Good catch, I didn't notice in the spreadsheet (and didn't look closely at the doc before) that these thresholds differed by league. This matches up to what I see in the doc 💯